django/contrib/gis/db/backends/postgis/operations.py

```diff
@@ -163,8 +158,8 @@ def __init__(self, connection):
         self.collect = prefix + 'Collect'
         self.difference = prefix + 'Difference'
         self.distance = prefix + 'Distance'
-        self.distance_sphere = prefix + 'distance_sphere'
-        self.distance_spheroid = prefix + 'distance_spheroid'
+        # self.distance_sphere =  ... see @cached_property below
+        # self.distance_spheroid =  ... see @cached_property below
```

Comment:
I've reverted all fixes related to GeoQuerySet.
Please take a look at the code, as I'm unsure about some of the stylistic choices I made.